python/cuml/tsa/holtwinters.pyx

```diff
@@ -484,7 +486,7 @@ class ExponentialSmoothing(Base):
                     raise IndexError("Index input: " + str(index) + " outside "
                                      "of range [0, " + str(self.ts_num) + "]")
                 else:
-                    return cudf.Series(self.level[index])
+                    return cudf.Series(cp.asarray(self.level[index]))
```

Comment:
does this function now need a decorator as it allocates cupy memory?

Reply:
it shouldn't be needed in this particular instance since self.level is an rmm device_array and this asarray is done no-copy so the allocator stays rmm. That said, overall holtwinters needs a somewhat significant cleanup to addopt cupy and cumlarray better (opening an issue in a bit)

Reply:
let's open it as an issue to adopt CumlArray here and address it then